src/http_util.rs

```diff
@@ -87,7 +97,8 @@ fn test_fetch_sync_string() {
 fn test_fetch_sync_string_with_redirect() {
   // Relies on external http server. See tools/http_server.py
   tokio_util::init(|| {
-    let p = fetch_sync_string("http://127.0.0.1:4546/package.json").unwrap();
+    let (p, _m) =
+      fetch_sync_string("http://127.0.0.1:4546/package.json").unwrap();
     println!("package.json len {}", p.len());
```

Comment:
Should you not test `_m` here?

Reply:
doh! forgot when I broke them to actually adjust them